_config.yml

```diff
@@ -20,12 +22,19 @@ defaults:
       layout: plugin              # DO NOT CHANGE THIS DEFAULT
       type: rubygems              # DO NOT CHANGE THIS DEFAULT
 
+  - scope:
+      type: themes
+    values:
+      permalink: /themes/:title/  # DO NOT CHANGE THIS DEFAULT
+      layout: theme               # DO NOT CHANGE THIS DEFAULT
+      type: rubygems              # DO NOT CHANGE THIS DEFAULT
```

Comment:
Idea: let's have a test spec that explicitly checks that the individual themes (and plugins above) abide by these conventions unless explicitly overridden in the case of `type` only.